README.md

```diff
@@ -43,9 +43,31 @@ Units of measurement can be formatted as follows:
 The syntax was discussed in #3.
 
 - `style` receives the string value "unit"
-- `unit` receives a string core unit identifier, defined in [UTS #35, Part 2, Section 6](http://unicode.org/reports/tr35/tr35-general.html#Unit_Elements).  See also the [full list of unit identifiers](https://unicode.org/repos/cldr/tags/latest/common/validity/unit.xml).
+- `unit` receives a string core unit identifier, defined in [UTS #35, Part 2, Section 6](http://unicode.org/reports/tr35/tr35-general.html#Unit_Elements).  A [subset](https://tc39.es/proposal-unified-intl-numberformat/section6/locales-currencies-tz_proposed_out.html#sec-issanctionedsimpleunitidentifier) of units from the [full list](https://unicode.org/repos/cldr/tags/latest/common/validity/unit.xml) was selected for use in ECMAScript; see a discussion on the methodology for choosing the subset in #39.
 - `unitDisplay`, named after the corresponding setting for currencies, `currencyDisplay`, takes either "narrow", "short", or "long".
 
+### Feature Detection: Measurement Units
+
+Check for a RangeError when passing `style: "unit"`:
+
+```javascript
+let numberFormat;
```

Comment:
I suggest you rewrite this part as a function return a boolean type. Such as 
```javascript
function IsUnitSupportedByIntlNumberFormat(unit) {
  try {
    let nf = new Intl.NumberFormat(undefined, {style: 'unit', unit});
  } catch (e) { 
    return false;
  }
  return true;
}
```

Reply:
Done